packages/types/src/interfaces.ts

```diff
@@ -13,5 +14,5 @@ export interface ApiWrapper {
   getFinalizedBlockHeight: () => Promise<number>;
   getBestBlockHeight: () => Promise<number>;
   getBlockByHeightOrHash: (hashOrHeight: number | string) => Promise<Block>;
-  fetchBlocks: (bufferBlocks: number[]) => Promise<EthereumBlock[] | LightEthereumBlock[]>; // TODO make sure this is correct
+  fetchBlocks: (bufferBlocks: number[]) => Promise<IBlock<EthereumBlock>[] | IBlock<LightEthereumBlock>[]>; // TODO make sure this is correct
```

Comment:
Im not sure why this function is exposed in the sandbox, maybe it needs to be removed